plejd/MqttClient.js

```diff
@@ -401,6 +446,11 @@ class MqttClient extends EventEmitter {
     // );
   }
 
+  buttonPressed(data) {
```

Comment:
Please add some typing for what "data" is expected to be `updateOutputState` might be good as inspiration

Reply:
Done

Reply:
Do push the code also 😉